### ISSUE TITLE
Javascript reference setSession docs are incorrect (recent change?)

### ISSUE THREAD

**Issue by GaryAustin1** [supabase/supabase-js#632]
# Improve documentation

## Link 
https://supabase.com/docs/reference/javascript/auth-setsession
![image](https://user-images.githubusercontent.com/54564956/200399985-698e07c1-c80d-4ed4-b6a5-25fbe55547bc.png)

## Describe the problem

The docs show using a refresh_token in two spots, but the parameter is listed as a session object and that was what they used to show I believe.

refreshSession() also seems strange...

## Describe the improvement

A clear and concise description of the improvement.

## Additional context

Add any other context or screenshots that help clarify your question.

**Comment by j4w8n** [comment]
I'm not sure why this hasn't been resolved. I guess the team is extremely busy. I'll write up a PR, but I'm unsure how soon it'll get merged.

I believe the contradiction on the page is because the parameters section comes from the code in the gotrue-js repo, but the rest of the page comes from code in the supabase repo.

As for refreshSession, it is correct. What is strange about it? Maybe something needs clarified.

**Comment by GaryAustin1** [comment]
@j4w8n Yeah, I did not pursue fixing in the docs as it got moved from there and it seems like several of the reference docs have been redone with error or missing info that existed just weeks ago in the v2 docs.  So I worried it was being done some other way as valid info was being thrown out like the process had changed and not just moving the old.

On refreshSession, I missed the brackets around refresh_session and am used to seeing objects spelling out in the docs like {refresh_session:refresh_session} but understand what they mean on closer look.

**Comment by mandarini** [comment]
This documentation has been updated and now correctly shows that `setSession()` accepts an object with `access_token` and `refresh_token` properties. The current documentation at https://supabase.com/docs/reference/javascript/auth-setsession accurately reflects the API. Closing as resolved. Thank you for contributing!
